### DISCUSSION TITLE
Reenabling WebGL2 by default on mobile

### DISCUSSION THREAD

**Discussion by arthuro555** [pixijs/pixijs#8452] [Ideas]
Hello 👋 

I was taking a peek at the `PIXI.settings` documentation when I saw [in PREFER_ENV that WebGL2 is not enabled on mobile devices by default](https://github.com/pixijs/pixijs/blob/db824181dc3dfffa0951fb8579282ba8e654cb3b/packages/core/src/settings.ts#L5-L20). I have taken a look at the referenced bug, and it is marked as fixed on the chrome bug tracker. It also has been about 3 years since that change was introduced.

I haven't tested the stability on mobile myself, but it looks like it might be time to look into the new state of WebGL2 on mobile today, and maybe reenable it by default everywhere.

**Comment by bigtimebuddy** [comment]
You're right, a change for this has been merged and will be added in v7 to always default to WebGL2 for all devices. It's a breaking change so it hasn't been changed in v6 yet.
